src/resolver/package_json.zig

```diff
@@ -667,6 +669,22 @@ pub const PackageJSON = struct {
             }
         }
 
+        if (json.asProperty("config")) |npm_pkg_cfg| {
```

Comment:
This should not be exposed in the runtime, it should have a flag to only expose it in `bun run`. Similar to what we do with the `"bin"` field

Reply:
(This way we don't copy all these config objects for things which will never use it, like the bundler)